make.py

```diff
@@ -98,6 +100,11 @@ def get_architecture(compiler, cpu):
 		if compiler == 'vs2017':
 			if cpu == 'arm64':
 				return 'ARM64'
+		elif compiler == 'vs2019':
+			if cpu == 'x86':
+				return 'Win32'
+			else:
+				return cpu
```

Comment:
For x64 and arm64 we just pass that string as-is?

Reply:
Yup, they work as is, so I didn't see the point in adding cases for them.